docs/faq/differences-with.../fivetran-vs-airbyte.md

```diff
@@ -0,0 +1,27 @@
+# Fivetran vs Airbyte
+
+We wrote an article, “[Open-source vs. Commercial Software: How to Solve the Data Integration Problem](https://airbyte.io/articles/data-engineering-thoughts/open-source-vs-commercial-software-how-to-better-solve-data-integration/),” in which we describe the pros and cons of Fivetran’s commercial approach and Airbyte’s open-source approach. Don’t hesitate to check it out for more detailed arguments. As a summary, here are the differences: 
+
+![](https://lh4.googleusercontent.com/bcNM7lS3u9ijFkQzJJoIIUUBzqYSnlpmBQWArPaBQOc2_1ztmDVeH-51wcK43zNWWOmLW-_SjIajenHPbejjorNrTmI31-v8r9h0gOLiqDXwVmmKKASea4ks2l-Y5BoxuFpWT7cz)
```

Comment:
We should't use external links like that. Can you add the image to gitbook and link to it?

Reply:
I can add the image to our wordpress and link to it. Is that ok? The external link was added by Gitbook automatically, so I think that's the way they do it